lib/widgets/story_video.dart

```diff
@@ -104,6 +108,7 @@ class StoryVideoState extends State<StoryVideo> {
             if (playbackState == PlaybackState.pause) {
               playerController!.pause();
```

Comment:
So What happens if you pause the video? Does it mean you can't mute volume even if it's paused?